app/src/main.cpp

```diff
@@ -0,0 +1,61 @@
+#include <QGuiApplication>
+#include <QQmlApplicationEngine>
+
+#include <QLocale>
+#include <QTranslator>
+
+#include <Helpers/helpers.h>
+#include <Helpers/logs.h>
+
+#include <QDir>
+#include <QDebug>
+
+using namespace Status;
+
+void setApplicationInformation(QGuiApplication& app);
+
+int main(int argc, char *argv[])
+{
+    //qInstallMessageHandler(Helpers::logFormatter);
+
+    QGuiApplication app(argc, argv);
+    
+    setApplicationInformation(app);
+  
+    QTranslator translator;
```

Comment:
Later i believe we can wrap translator with controller to access and switch locale from QML

Reply:
Please consider it when dealing with the specific task: https://github.com/status-im/status-desktop/issues/5689.